pkg/cmd/run.go

```diff
@@ -360,7 +361,8 @@ func init() {
 	runCmd.Flags().Bool("fallback-readonly", false, "disable modifying the fallback file. secrets can still be read from the file.")
 	runCmd.Flags().Bool("fallback-only", false, "read all secrets directly from the fallback file, without contacting Doppler. secrets will not be updated. (implies --fallback-readonly)")
 	runCmd.Flags().Bool("no-exit-on-write-failure", false, "do not exit if unable to write the fallback file")
-	runCmd.Flags().Bool("silent", false, "disable error output if the supplied command exits non-zero")
+	runCmd.Flags().Bool("silent", false, "disable output of info messages")
+	runCmd.Flags().Bool("silent-exit", false, "disable error output if the supplied command exits non-zero")
```

Comment:
I like this!

Reply:
This is actually a breaking change. But since this was just released two days ago and only one customer is using it, I think it's alright. I'll notify them directly.